packages/apps/frontera/src/routes/src/components/RootSidenav/RootSidenav.tsx

```diff
@@ -51,7 +50,7 @@ export const RootSidenav = observer(() => {
         handleItemClick={handleItemClick}
         togglePreference={togglePreference}
       />
-      <UserActionSection />
+      {/* <UserActionSection /> */}
```

Comment:
why it's commented out? will it be restored?

Reply:
steyn wasn't sure we gonna added back or not so i left commented